<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="7,1,50">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2,1,2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_load">
         <property name="text">
          <string>Vehicle</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_condition">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_simulate">
         <property name="text">
          <string>Analysis</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_load">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>781</width>
           <height>341</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_vehiclename" stretch="1,3,0,6">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Vehicle name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_loadfile">
              <property name="text">
               <string>Load Vehicle File</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_editvars" stretch="6,10">
            <item>
             <widget class="QGroupBox" name="groupBox_vars">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Edit variables</string>
              </property>
              <widget class="QWidget" name="verticalLayoutWidget">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>261</width>
                 <height>281</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_loadFile">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
               </layout>
              </widget>
              <zorder>verticalLayoutWidget</zorder>
              <zorder>line</zorder>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_display" stretch="6,1">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QTableView" name="tableView_generaldata"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_sql">
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_undo">
                  <property name="text">
                   <string>Undo</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_confirm">
                  <property name="text">
                   <string>Confirm</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_conditions">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Test Mode</string>
         </property>
        </widget>
        <widget class="Line" name="line_3">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>0</y>
           <width>20</width>
           <height>371</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_simulation"/>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>line</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
